src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  // write your code here0
+  constructor(name, health = 100, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
+    this.health = health;
```

Comment:
The same applies to the `Herbivore` class. You don't need to set `this.health = health;` again in the constructor. The `super(name, health);` call in the previous line already sets the health property to the passed value or the default value of 100. This is a code quality issue.